apps/marketplace/components/validators.js

```diff
@@ -79,6 +79,21 @@ export const validEmail = val => {
   return false
 }
 
+export const validGovernmentEmail = (emailAddress, userEmailAddress) => {
+  if (!validEmail(emailAddress)) {
```

Comment:
In RC i'm failing the `Name and email` stage against this validation when I don't provide a team address. I think the problem is we've expecting an empty value to fail `validEmail()` here, but in fact an empty value passes this validation. I think we'll need a dedicated `if (!emailAddress) return true` here above this.